mkdocs.yml

```
@@ -0,0 +1,71 @@
+
+markdown_extensions:
+  - admonition
+  - attr_list
+  - pymdownx.details
+  - pymdownx.superfences
+  - pymdownx.superfences
+  - pymdownx.tabbed:
+      alternate_style: true
+
+nav:
+  - Getting Started: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)